Utility/treeHelper/ui_treewindow.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+# Form implementation generated from reading ui file 'tree_window_.ui'
+#
+# Created by: PyQt5 UI code generator 5.15.7
+#
+# WARNING: Any manual changes made to this file will be lost when pyuic5 is
+# run again.  Do not edit this file unless you know what you are doing.
+
+
+from PySide2 import QtCore, QtGui, QtWidgets

```

Comment:
Please upload the ui file and replace with `gui.PySideUic.loadUiType('relative_path_to_ui_file')`. Cf. https://github.com/FreeCAD/FreeCAD-macros/wiki/Dynamically-load-graphical-interface-definition.